cross/libudev/Makefile

```diff
@@ -0,0 +1,97 @@
+PKG_NAME = libudev
+PKG_REAL_NAME = systemd
+PKG_VERS = 204
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/systemd/systemd/archive
+PKG_DIST_FILE = $(PKG_REAL_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIR = $(PKG_REAL_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/libgcrypt cross/dbus
+OPTIONAL_DEPENDS = cross/libcap cross/libcap_2.51
+
+# unable to find "library containing cap_init..."
+UNSUPPORTED_ARCHS = $(OLD_PPC_ARCHS)
+
+
+HOMEPAGE = https://github.com/systemd/systemd
+COMMENT = The systemd System and Service Manager provides an array of system components for Linux operating systems.
+# we take libraries only (tools are GPLv2)
+LICENSE = LGPLv2.1
+
+include ../../mk/spksrc.archs.mk
+ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(OLD_PPC_ARCHS)),$(ARCH))
+DEPENDS += cross/libcap_2.51
+else
+DEPENDS += cross/libcap
+endif
+
+GNU_CONFIGURE = 1
+PRE_CONFIGURE_TARGET = udev_pre_configure
+POST_CONFIGURE_TARGET = udev_post_configure
+POST_INSTALL_TARGET = udev_post_install
+
+# disable as much as possible...
+CONFIGURE_ARGS  = --without-python
+CONFIGURE_ARGS += --disable-manpages
+CONFIGURE_ARGS += --disable-tests
+CONFIGURE_ARGS += --disable-gudev
+CONFIGURE_ARGS += --disable-ima
+CONFIGURE_ARGS += --disable-chkconfig
+CONFIGURE_ARGS += --disable-selinux
+CONFIGURE_ARGS += --disable-xz
+CONFIGURE_ARGS += --disable-tcpwrap
+CONFIGURE_ARGS += --disable-pam
+CONFIGURE_ARGS += --disable-acl
+CONFIGURE_ARGS += --disable-xattr
+CONFIGURE_ARGS += --disable-gcrypt
+CONFIGURE_ARGS += --disable-audit
+CONFIGURE_ARGS += --disable-libcryptsetup
+CONFIGURE_ARGS += --disable-qrencode
+CONFIGURE_ARGS += --disable-microhttpd
+CONFIGURE_ARGS += --disable-binfmt
+CONFIGURE_ARGS += --disable-vconsole
+CONFIGURE_ARGS += --disable-readahead
+CONFIGURE_ARGS += --disable-bootchart
+CONFIGURE_ARGS += --disable-quotacheck
+CONFIGURE_ARGS += --disable-randomseed
+CONFIGURE_ARGS += --disable-logind
+CONFIGURE_ARGS += --disable-hostnamed
+CONFIGURE_ARGS += --disable-timedated
+CONFIGURE_ARGS += --disable-localed
+CONFIGURE_ARGS += --disable-coredump
+CONFIGURE_ARGS += --disable-polkit
+CONFIGURE_ARGS += --disable-efi
+CONFIGURE_ARGS += --disable-myhostname
+CONFIGURE_ARGS += --disable-keymap
+CONFIGURE_ARGS += --with-dbuspolicydir=$(INSTALL_PREFIX)/share/dbus-1/system.d
+CONFIGURE_ARGS += --with-dbussessionservicedir=$(INSTALL_PREFIX)/share/dbus-1/services
+CONFIGURE_ARGS += --with-dbussystemservicedir=$(INSTALL_PREFIX)/share/dbus-1/system-services
```

Comment:
Opened question: should theses aligne with actual DSM install path and therefore rely on them?

DSM6 (And what about the non-existing ones, probably due to older version in use?) :
```
/usr/share/dbus-1/services
/usr/share/dbus-1/system-services
```
DSM7 (still missing `interfaces` directory):
```
/usr/share/dbus-1/services
/usr/share/dbus-1/system-services
/usr/share/dbus-1/system.d
```

Reply:
> EDIT: Further investigating I believe `interfaces` is restricted to desktop components (e.g. `org.freedesktop.*` , `org.gnome.Shell.*`, `org.kde.*`) and is probably not needed at all.

interfaces directory is not supported/configurable in systemd-219 anymore.

Reply:
Question really is, currently we are making theses directories to point to the package path, but should it instead point to the system-wide default dbus-1 directories?  While I do not have a definitive answer on this I'd recommend using the system-wide path, similar to openssl certificates.